app/routers/session_answers.py

```diff
@@ -83,6 +84,84 @@ async def update_session_answer_in_a_session(
     return JSONResponse(status_code=status.HTTP_200_OK, content=None)
 
 
+@router.patch("/{session_id}", response_model=None)
+async def update_all_session_answers_in_a_session(
+    session_id: str, session_answers: List[UpdateSessionAnswer]
+):
+    """
+    Update all session answers in a session in the session answers array
+    Path Params:
+    session_id - the id of the session
+
+    function params:
+    session_answers - list of session answers for the session
+    """
+    log_message = f"Updating all session answers for session: {session_id}"
+    input_session_answers = [
+        jsonable_encoder(remove_optional_unset_args(session_answer))
+        for session_answer in session_answers
+    ]
+
+    # check if the session exists
+    session = client.quiz.sessions.find_one({"_id": session_id})
+    if session is None:
+        logger.error(
+            f"Received all session_answer update request, but provided session with id {session_id} not found"
+        )
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail=f"Received all session_answer update request, but provided session with id {session_id} not found",
```

Comment:
you can save this in a variable and use it in the logger.error above and here as well. No need to write the log statements strings again